code/control/DNRoot.php

```diff
@@ -1096,14 +1096,6 @@ public function gitRevisions(SS_HTTPRequest $request) {
 			return $this->environment404Response();
 		}
 
-		// For now only permit advanced options on one environment type, because we hacked the "full-deploy"
```

Comment:
yeah, I think I need to add something so each environment has a `supported_advanced_features` or something. Any suggestions?